RetailCoder.VBE/Inspections/InspectionBase.cs

```diff
@@ -81,6 +83,28 @@ protected virtual IEnumerable<Declaration> BuiltInDeclarations
             get { return State.AllDeclarations.Where(declaration => declaration.IsBuiltIn); }
         }
 
+        protected bool HasIgnoreAnnotation(VBComponent component, int line)
```

Comment:
Could they be combined/consolidated somehow? I don't like scattering that responsibility. I like having it here in `InspectionBase`, seems to fit right in.